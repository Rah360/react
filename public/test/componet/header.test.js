import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Dashboardpage'

test('testing header componets',()=>{
    const renderer=new ReactShallowRenderer()
    renderer.render(<Header/>)
    console.log(renderer.getRenderOutput())
})
 