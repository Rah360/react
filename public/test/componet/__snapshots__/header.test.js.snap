// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header componets 1`] = `
<div>
  <h3>
    DashBoard Page
  </h3>
  <hr />
  <hr />
  <Connect(expenseList) />
</div>
`;
